{"name": "test_get_users_with_incorrect_limit_017", "status": "skipped", "statusDetails": {"message": "Skipped: {id записи об ошибке} Вместо 422 получаем статус-код 200. Skip-аем пока не починят", "trace": "('F:\\\\PYTHON PROJECTS\\\\RESTAPI_TECH\\\\tests\\\\users\\\\test_users.py', 45, 'Skipped: {id записи об ошибке} Вместо 422 получаем статус-код 200. Skip-аем пока не починят')"}, "description": "\n    Получить список пользователей с отрицательным query-параметрам limit = -1\n\n    Ожидаемый результат:\n        Статус-код 422;\n        Время ответа сервера - не превышает 500ms;\n        Схема JSON-ответа соответствует Требованиям;\n        Response header \"Content-Type\": \"application/json\"\n        Response header \"Connection\": \"keep-alive\"\n        В JSON присутствует описание ошибки\n\n    Полученный результат: Выгружены все пользователи из БД, статус-код 200\n    ", "start": 1721415263921, "stop": 1721415263921, "uuid": "3b837ec6-4884-4c69-be18-c92ac56326cc", "historyId": "531f30cd2268d6ad88b8a38497c1393f", "testCaseId": "531f30cd2268d6ad88b8a38497c1393f", "fullName": "tests.users.test_users#test_get_users_with_incorrect_limit_017", "labels": [{"name": "tag", "value": "users"}, {"name": "tag", "value": "@pytest.mark.skip('{id записи об ошибке} Вместо 422 получаем статус-код 200. Skip-аем пока не починят')"}, {"name": "parentSuite", "value": "tests.users"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "DESKTOP-55MLA3F"}, {"name": "thread", "value": "23412-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.users.test_users"}]}